package.json

```diff
@@ -80,12 +80,12 @@
   "dependencies": {
     "@octokit/rest": "^17.1.4",
     "async": "2.1.2",
-    "auth0": "^2.23.0",
+    "auth0": "^2.27.0",
     "auth0-extension-express-tools": "^2.1.0",
     "auth0-extension-tools": "1.4.5",
     "auth0-extension-ui": "^1.0.1",
     "auth0-oauth2-express": "^1.1.8",
-    "auth0-source-control-extension-tools": "^4.0.5",
+    "auth0-source-control-extension-tools": "^4.1.0",
```

Comment:
TODO: bump to 4.1.1 when https://github.com/auth0-extensions/auth0-source-control-extension-tools/pull/99/ is merged.